currency_rate_update_VN_VCB/tests/test_currency_rate_update_vn_vcb.py

```diff
@@ -0,0 +1,47 @@
+# © 2018 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestCurrencyRateUpdateVnVcb(common.SavepointCase):
+
+    def setUp(self):
+        super(TestCurrencyRateUpdateVnVcb, self).setUp()
+        self.usd = self.env.ref('base.USD')
+        self.euro = self.env.ref('base.EUR')
+        self.vnd = self.env.ref('base.VND')
+        self.vnd.write({'active': True})
+        currency_rates = self.env['res.currency.rate'].search(
+            [('currency_id', '=', self.usd.id)])
+        currency_rates.unlink()
+        currency_rates = self.env['res.currency.rate'].search(
+            [('currency_id', '=', self.euro.id)])
+        currency_rates.unlink()
+        currency_rates = self.env['res.currency.rate'].search(
+            [('currency_id', '=', self.vnd.id)])
+        currency_rates.unlink()
+        self.main_currency = self.env.user.company_id.currency_id
+        self.update_service = self.env['currency.rate.update.service'].create({
+            'service': 'VN_VCB',
+            'currency_to_update': [(6, 0,
+                                    [self.euro.id, self.vnd.id, self.usd.id])]
+        })
+
+    def test_currency_rate_update_USD_VND(self):
```

Comment:
Keep only one function `check_currency_rate_update_base_currency` 
```
self.assertEqual(len(self.eur.rate_ids), 0)
self.assertTrue(len(self.usd.rate_ids) > 0)
self.assertTrue(len(self.vnd.rate_ids) > 0)
```
One more function where we change the company's currency: `check_currency_rate_update_new_currency`
```
self.update_service.company_id.currency_id = self.vnd.id
self.assertTrue(len(self.eur.rate_ids) > 0)
self.assertTrue(len(self.usd.rate_ids) > 0)
self.assertEqual(len(self.vnd.rate_ids), 0)
```